source/testnet/guides/run-node.rst

```diff
@@ -34,7 +34,7 @@ Before running a Concordium node you will need to
 
    -  On *Linux*, allow Docker to be run as a non-root user.
 
-2. Download and extract the `Concordium Node and Client`_ software. This
+2. Download and extract the :ref:`concordium-node-and-client-download` software. This
    produces a directory named ``concordium-software``. Move this folder
    into the ``Documents`` folder of your home directory. It's very
    important that this directory is moved in its entirety and with the
```

Comment:
This is no longer the case. They can unpack it anywhere they want.

So everything after "This produces ..." in this paragraph is redundant.